pkgs/development/python-modules/galois/default.nix

```diff
@@ -1,5 +1,6 @@
 { lib
 , buildPythonPackage
+, setuptools_scm
```

Comment:
I think this has been renamed to `setuptools-scm`?

```
error: Function called without required argument "setuptools_scm" at ...
```

Reply:
Both `setuptools-scm` and `setuptools_scm` work. Is this something that will be deprecated in the future? I renamed it here

Reply:
Yes, aliases are the way to deprecate attributes. There is an `allowAliases` switch in `nixpkgs.config` that one can use to disallow aliases and notice these breakages earlier.

Reply:
Awesome, I will use that in the future

Reply:
:thinking: are you testing your bare branch rather than the merged result with master?

Reply:
Turns out, I was testing on a ~3 week old branch instead of the latest

Reply:
I would have thought ofborg would have caught that error? Definitely a lesson learned. Sorry about that